### PR TITLE
Feature accessible

### PR DESCRIPTION
**Pull request by Tchilo**
Easy Checks  Web Accessibility Report:

- Page titles - no issues found
- Image text alternatives** (alt text) - no issues found
- Text headings - see inline comment
- Color contrast - see inline comment
- Resize  - no issues found
- Interaction - no issues found
- Moving content - no issues found
- Multimedia  - no issues found
- The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -174,7 +174,7 @@ <h2 class="me">About me</h2>
     <ul class="about-myself">
       <li>
         <img src="images/icons/diamond.png" alt="orange diamond shape">
-        <p class="my-skills">Language</p>
```
Comment (Tchilo) [inline index.html:177]: possible heading, change from <p> to <h3>

File: index.html [hunk 2]
```diff
@@ -184,7 +184,7 @@ <h2 class="me">About me</h2>
       </li>
       <li>
         <img src="images/icons/rec.png" alt="orange square shape">
-        <p class="my-skills">Frameworks</p>
+        <h3 class="my-skills">Frameworks</h3>
```
Comment (Tchilo) [inline index.html:188]: possible heading, change from <p> to <h3>

File: index.html [hunk 3]
```diff
@@ -195,7 +195,7 @@ <h2 class="me">About me</h2>
       </li>
       <li>
         <img src="images/icons/circle.png" alt="orange circle shape">
-        <p class="my-skills">Skills</p>
```
Comment (Tchilo) [inline index.html:198]: possible heading, change from <p> to <h3>

File: style.css [hunk 4]
```diff
@@ -362,8 +362,9 @@ body {
   margin: 8px;
 }
 
-.about-myself p {
```
Comment (Tchilo) [inline style.css:365]: possible heading, change from <p> to <h3>

File: index.html [hunk 5]
```diff
@@ -211,9 +212,14 @@ <h3>I'm always interested in
...
-          <input class="email" type="email" name="email-address" placeholder="Email address" required>
-          <textarea class="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Enter text here"
+          <label for="fname">Full name:</label>
+          <input id="fname" type="text" maxlength="30" name="full-name" placeholder="Full name" required>
+
+          <label for="email">Email</label>
+          <input id="email" type="email" name="email-address" placeholder="Email address" required>
+
+          <label for="textarea">Message:</label>
+          <textarea id="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Enter text here"
```
Comment (Tchilo) [inline index.html:222]: Add label tag

File: index.html [hunk 6]
```diff
@@ -222,19 +228,19 @@ <h3>I'm always interested in
...
-          <a href="#"><i class="fa fa-angellist clr-secondary"></i></a>
+          <a href="#" aria-label= "angellist"><i class="fa fa-angellist clr-secondary"></i></a>
         </li>
         <li>
-          <a href="#"><i class="fa fa-twitter"></i></a>
+          <a href="#" aria-label= "twitter"><i class="fa fa-twitter"></i></a>
         </li>
         <li>
-          <a href="#"><i class="fa fa-medium"></i></a>
+          <a href="#" aria-label= "medium"><i class="fa fa-medium"></i></a>
```
Comment (Tchilo) [inline index.html:243]: Add aria-lable

File: style.css [hunk 7]
```diff
@@ -685,6 +695,11 @@ input::placeholder {
   .list3,
   .list6 {
     background-image: url(images/no4.png);
+    filter: contrast(1.75);
+  }
+
+  .list4 {
+    filter: contrast(1.4);
```
Comment (Tchilo) [inline style.css:702]: Add contrast to background

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Tchilo,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**